### ISSUE TITLE
User module forms error messages are not translated

### ISSUE THREAD

**Issue by jptissot** [OrchardCMS/OrchardCore#5086]
When logging in / registering and an error occurs, the messages displayed are not translated.

![image](https://user-images.githubusercontent.com/4681586/71199198-6e8c0c00-2263-11ea-9207-322cf35de6fd.png)

**Comment by Skrypt** [comment]
Related with #4675, #5050

**Comment by hishamco** [comment]
This is a known issue as @Skrypt refer to

**Comment by hishamco** [comment]
#5050 is not related, it was to fix another issue that may related to data annotation, and it's ready to merge

**Comment by jptissot** [comment]
Another issue I found. When you enable the ASP.Net option to disable the registration of duplicate emails, the error message is strange and not translated as well.

**Comment by hishamco** [comment]
The issue mainly with data annotations if I'm not wrong

**Comment by hishamco** [comment]
@jptissot did you add a `*.po` file that contains the error messages for the French translations?

**Comment by jptissot** [comment]
I did not.

**Comment by hishamco** [comment]
So, how the messages will got localized?!! Have a look to this one https://github.com/aspnet/Entropy/blob/master/samples/Localization.StarterWeb/Resources/ViewModels/Account/RegisterViewModel.fr.resx as sample, seems we all are forgot to set the localized messages :)

**Comment by jptissot** [comment]
I mean, I installed the *.po files from Crowdin but those strings are not located in those files, unless I am mistaken.

We also need to think about the password requirements which are set on the startup. 
For example: 
```
  services.Configure<IdentityOptions>(options =>
            {
               options.Password.RequiredLength = 8;
               options.Password.RequiredUniqueChars = 3;
               options.User.RequireUniqueEmail = true;
            }); 
```

The RequireUniqueEmail setting yields this error message when a user tries to register with the same email: 

![image](https://user-images.githubusercontent.com/4681586/71366983-91733480-2571-11ea-9435-0b9e2f18c581.png)

**Comment by hishamco** [comment]
Please hold on Jean, I'm on the final step to fix the data annotation issue, I will keep you updated on this

**Comment by hishamco** [comment]
FYI @jptissot I just updated my PR, so you could have a try

**Comment by agriffard** [comment]
Can this be closed @jptissot  ?

**Comment by hishamco** [comment]
I think we can close this while there's another issue related to the PR

**Comment by jptissot** [comment]
Which pr ? I did not get a chance to test this !

**Comment by agriffard** [comment]
@jptissot  #5416

**Comment by Skrypt** [comment]
![image](https://user-images.githubusercontent.com/3228637/74691677-c1296c00-51b1-11ea-8412-7df8169457a7.png)

**Comment by agriffard** [comment]
@hishamco I fixed the conflicts in #4675 and tried it but the validation messages are not localized.
Do we have another way to fix this?

If not, we will have to remove the Required attributes in the model, validate that the properties are set in the driver and add a T[""] error message to the ModelState if they are empty.

**Comment by hishamco** [comment]
Let me check the PR again, it has been awhile, hopefully I will have a look today

Thanks
